cms/envs/production.py

```diff
@@ -551,3 +551,38 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+DJANGO_DEBUG = ENV_TOKENS.get('DJANGO_DEBUG', False)
```

Comment:
Do we need to set debug=True for dev server?

Reply:
DJANGO_DEBUG is an env variable that I added for dev so we don't add extra middleware on stage and prod.

Reply:
I have renamed it to `ACTIVE_DJANGO_DEBUG_TOOLBAR` on suggestion.